### ISSUE TITLE
direction of ditches/drains

### ISSUE THREAD

**Issue by Mahabarata** [openstreetmap/iD#9665]
### Description

Hi,

Currently, when a ditch flowing in a direction is connected to another ditch flowing in another direction (from the common node), we have a warning in iD.

But in the reality, we have a lot of ditches connected at a point (they look like one ditch) with water going from that point to 2 different directions because the point is a high point.

I assume there is the same for the drains although I haven't seen any yet.

Maybe the warning could be removed for these 2 types of waterways.

Best regards

**Comment by Dimitar5555** [comment]
You can add `oneway=no` to the segment that represents the high point.

**Comment by Mahabarata** [comment]
It doesn't work on a node.

**Comment by tyrasd** [comment]
@Mahabarata an example of such a situation would be useful to reproduce the issue.

From what you describe, I think these situations are quite rare in practice. And the workaround to mark a short section of way (on the flat part on top of the "hill") as `oneway=no` could do the trick to pacify the validator (and it's probably the most accurate description of the ditch as the water would be able to flow in both directions on that part of the ditch).

**Comment by Mahabarata** [comment]
Here an example :
https://www.openstreetmap.org/edit?node=12844129124#map=20/4.8949030/-52.3296740

There is a ditch all along the parking with 2 outlets : when you follow the parking the 2 inclines are clearly visible.
The point is at the watershed so it is a point, it is not a way where the rain water, according of its "mood" (bipolaire water ? lol) decides one day to go to the right and another day to the left : for me oneway=no is only for rivers in the tide area.

Of course another solution could be to separate the ways but it it not the truth because there is one ditch with 2 inclines, not 2 ditches.

**Comment by tordans** [comment]
I agree with how it is mapped in this example. 

The question is what to do about the warning. 
- remove it for ditch … but that could lead to more routing mistakes. I don’t think that is an improvement 
- find a rag for the node that helps the validator and can be used to fix the warning — needs discussion elsewhere
- use the „ignore this“ option in the warning and keep all as is. 
- ?

**Comment by Mahabarata** [comment]
instead of oneway=no, it could be waterway=watershed on the node : currently this tag is used wrongly in osm database, most are used on relations type=waterway instead of type=watershed.
waterway on a node is not a bad tag (we have waterfalls, weirs, rapids... on nodes) and the meaning of waterway=watershed is clear for everyone I think.

**Comment by tyrasd** [comment]
> according of its "mood" (bipolaire water ? lol) 

😅  well, the point where the water shed is located would depend on external factors like the wind, etc. so, IMHO it would be fine to use the `oneway=no` trick to map such ditches. But I would agree that it seems a bit like a hacky way to map this.

> `waterway=watershed` on the node

That would be an elegant option, but unfortunately that tag is not documented on the wiki, and is only used [once](https://taginfo.openstreetmap.org/tags/waterway=watershed?filter=nodes#overview) on nodes.

So, to summarize, it seems like there does not yet exist a proper way to tag such situations. That means, the issue tracker for iD is not the correct place to discuss this. Instead it would be better to start a discussion in the OSM community about this specific tagging question. Ideally you could propose a possible solution, e.g. the one with the `watershed` tag on nodes.

**Comment by emil813-cmd** [comment]
The editor has been made to allow two ditches to emerge from the same natural=spring, so your solution only needs for that code to also apply to the watershed.

But the word [watershed](https://duckduckgo.com/?t=ftsa&q=watershed&ia=web) has too many contradictory meanings. I think you should use a different name like "divergence_point", "saddle_point", etc.

**Comment by matkoniecz** [comment]
We first need agreement from OSM community how to mark such places (if there is truly no interrupted ditch there).

Is anyone interested in starting discussion? Maybe via making a tagging proposal?

**Comment by matkoniecz** [comment]
first tagging for such case needs to be decided by OSM community, probably at https://community.openstreetmap.org/c/general/tagging/70 or tagging mailing list

**Comment by matkoniecz** [comment]
once that happens new issue can be opened (or this requested to be reopened, but making new one and linking this one will be more reliable)
